micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/InstrumentedQueuedThreadPool.java

```diff
@@ -116,4 +120,18 @@ protected void doStart() throws Exception {
         threadPoolMetrics.bindTo(registry);
     }
 
+    @Override
+    protected void doStop() throws Exception {
+        Set<Tag> tagSet = new HashSet<>();
+        tags.forEach(tagSet::add);
+        Stream
+            .of("jetty.threads.config.min", "jetty.threads.config.max", "jetty.threads.busy", "jetty.threads.jobs",
+                    "jetty.threads.current", "jetty.threads.idle")
```

Comment:
I worry about keeping this list in sync with changes in `JettyServerThreadPoolMetrics`. It would be better to keep track of the metrics in `JettyServerThreadPoolMetrics` and make it `AutoCloseable`. Then `doStop` here just needs to close its instance of `JettyServerThreadPoolMetrics`. You can see an example of where we have done this in `KafkaMetrics`.